examples/config.hocon.sample

```diff
@@ -55,8 +55,10 @@ aws {
   secretKey = iam
 }
 
-# config for NSQ
-nsq {
+queueConfig {
```

Comment:
We use `source` in stream enrich, so I think we should try to keep things consistent here too

Reply:
ah, just saw your remark in #68, we went with `sourceSink` in stream enrich